graylog2-server/src/main/java/org/graylog2/inputs/transports/AbstractHttpTransport.java

```diff
@@ -173,6 +197,10 @@ public ConfigurationRequest getRequestedConfiguration() {
                     "Enable Bulk Receiving",
                     false,
                     "Enables bulk receiving of messages separated by newlines (\\n or \\r\\n)"));
+            r.addField(new BooleanField(CK_ENABLE_JSON_ARRAY_PROCESSING,
+                    "Enable JSON Array Processing",
+                    false,
+                    "Enables processing of JSON arrays by extracting individual JSON objects from the array."));
```

Comment:
If we do remove the config option, we should add a note about JSON arrays being supported in the line above. Something like `"Enables bulk receiving of messages separated by newlines (\\n or \\r\\n) or in JSON arrays"`